app/scripts/modules/core/src/filterModel/FilterCollapse.less

```diff
@@ -1,10 +1,41 @@
 .filters-toggle {
   position: absolute;
 
-  .filters-placeholder {
+  .filters-hidden {
     margin-top: 11px;
   }
 
+  .filters-open {
+    background-color: var(--color-accent-g2);
+    height: 32px;
+    font-size: 16px;
+    margin-top: 6px;
+
+    @media (min-width: 768px) and (max-width: 992px) {
+      width: 104px;
+    }
+
+    @media (min-width: 992px) {
```

Comment:
this is a lot of breakpoints.  If it's trying to utilize more space as the screen gets wider, could we use flexbox instead to get a nice smooth transition?  

Also, do we have CSS variables for breakpoint widths?  If not, we probably should.

Reply:
we currently only have a mobile breakpoint. I can set up a couple more for s, m, l, etc screens as well.